<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportWidget</class>
 <widget class="QWidget" name="ReportWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Report</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="form">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Code</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditCode">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Language/Country</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="RelationalComboBox" name="comboBoxL10n">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="editable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Class</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QComboBox" name="comboBoxClass"/>
           </item>
           <item row="0" column="6">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>168</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="7">
            <widget class="QCheckBox" name="checkBoxSystem">
             <property name="focusPolicy">
              <enum>Qt::FocusPolicy::NoFocus</enum>
             </property>
             <property name="text">
              <string>System report</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Description</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="7">
            <widget class="QLineEdit" name="lineEditDescription"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>XML report definition</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPlainTextEdit" name="textEditXML">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="view">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="EnhancedTableView" name="tableView"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Font</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFontComboBox" name="fontComboBox">
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="maxVisibleItems">
          <number>20</number>
         </property>
         <property name="fontFilters">
          <set>QFontComboBox::FontFilter::MonospacedFonts</set>
         </property>
         <property name="currentFont">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxFontSize"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonInsertImage">
         <property name="text">
          <string>Insert image to clipboard</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonDeleteAll">
         <property name="text">
          <string>Delete All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonDownload">
         <property name="text">
          <string>Download ...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonUpload">
         <property name="text">
          <string>Upload ...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonDownloadAll">
         <property name="text">
          <string>Download all reports...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonUploadAll">
         <property name="text">
          <string>Upload all reports ...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EnhancedTableView</class>
   <extends>QTableView</extends>
   <header>App.Widget.View</header>
  </customwidget>
  <customwidget>
   <class>RelationalComboBox</class>
   <extends>QComboBox</extends>
   <header>App.Widget.Control</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditCode</tabstop>
  <tabstop>comboBoxL10n</tabstop>
  <tabstop>comboBoxClass</tabstop>
  <tabstop>lineEditDescription</tabstop>
  <tabstop>textEditXML</tabstop>
  <tabstop>fontComboBox</tabstop>
  <tabstop>spinBoxFontSize</tabstop>
  <tabstop>pushButtonInsertImage</tabstop>
  <tabstop>pushButtonDownload</tabstop>
  <tabstop>pushButtonUpload</tabstop>
  <tabstop>pushButtonDownloadAll</tabstop>
  <tabstop>pushButtonUploadAll</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>checkBoxSystem</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
